### PR TITLE
{bio,lib}[GCCcore/10.3.0] dcm2niix v1.0.20211006, CharLS v2.3.4

### PR DESCRIPTION
**Pull request by bedroge**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/c9151002c2f186bdbecff83994c55a15 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/d/dcm2niix/dcm2niix-1.0.20211006-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'dcm2niix'
+version = '1.0.20211006'
+
+homepage = 'https://github.com/rordenlab/dcm2niix'
+description = """dcm2niix is a designed to convert neuroimaging data from the DICOM format to the NIfTI format."""
```
Comment (verdurin) [inline easybuild/easyconfigs/d/dcm2niix/dcm2niix-1.0.20211006-GCCcore-10.3.0.eb:7]:
```suggestion
description = """dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format."""
```

**Review by verdurin — CHANGES_REQUESTED** [review]

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @bedroge!

**Pull request merged**